app/controllers/api/exchange_products_controller.rb

```diff
@@ -9,6 +9,7 @@ class ExchangeProductsController < Api::BaseController
 
     skip_authorization_check only: [:index]
 
+    include PaginationData
```

Comment:
Minor style thing: I think I prefer includes consistently at the top of the class, just under the class declaration and before everything else.